src/libraries/Microsoft.Extensions.Hosting/src/Internal/HostingLoggerExtensions.cs

```diff
@@ -11,12 +11,11 @@ internal static class HostingLoggerExtensions
     {
         public static void ApplicationError(this ILogger logger, EventId eventId, string message, Exception exception)
         {
-            var reflectionTypeLoadException = exception as ReflectionTypeLoadException;
-            if (reflectionTypeLoadException != null)
+            if (exception is ReflectionTypeLoadException reflectionTypeLoadException)
             {
                 foreach (Exception ex in reflectionTypeLoadException.LoaderExceptions)
                 {
-                    message = message + Environment.NewLine + ex.Message;
+                    message = $"{message}{Environment.NewLine}{ex.Message}";
```

Comment:
This change is unnecessary. Can you revert it?

Note that they both compile into the exact same IL.